packages/backend/src/clients/Google/GoogleDriveClient.ts

```diff
@@ -52,4 +49,44 @@ export class GoogleDriveClient {
             console.debug(`${file.name} (${file.id})`);
         });
     }
+
+    async appendToSheet(
+        refreshToken: string,
+        fileId: string,
+        csvContent: string,
+    ) {
+        if (!this.isEnabled) {
+            throw new Error('Google Drive is not enabled');
+        }
+        const authClient = await GoogleDriveClient.getCredentials(refreshToken);
+
+        const sheets = google.sheets({ version: 'v4', auth: authClient });
+        const sheetRows = csvContent.split('\n').map((row) => row.split(','));
```

Comment:
I don't think this can be used in place of a csv-reader library. For example this will split quoted strings `","` in a single cell.

Reply:
I'm using `csvtojson` to convert the file into a `Record<string,string>[]` , that library even supports streaming. I don't see the advantage of writting a json format at the moment, and that requires a bit of additional refactoring . We can consider this on a separate ticket